index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="ro">
+    <meta charset="UTF-8">
+    
+    <head class="header">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title> Save the planet</title>
+        <link href="./CSS/homestyle.css" rel=stylesheet>         
+    </head>
+    <body>
+        <section class="coverContainer">
+            <div class="coverImage">
+                <div class="shape"><img  src="./Photos/Shape.png" alt="shape"></div>
+                <div class="coverContent">
+                    <div class="subtitle">
+                        <div class="ractangleImg"><img  src="./Photos/Rectangle.png" alt="rectangle"></div>
+                        WE TEACH YOU TO PROTECT OUR   
+                    </div>
+                    <div class="title">PLANET</div>
+                    <div class="description">
+                        Our planet is precious and vital for our<br>
+                        beautiful lives, if we don’t take care of it<br>
+                        it will dissapear along with us.
+                    </div>  
+                    <div class="mention">
```

Comment:
in toata sectiunea mergi cu clasele de hero
```suggestion
                    <div class="hero__mention">
```